crates/static-analysis-kernel/src/analysis/ddsa_lib/common.rs

```diff
@@ -13,8 +13,10 @@ pub type NodeId = u32;
 
 #[derive(Debug, thiserror::Error)]
 pub enum DDSAJsRuntimeError {
-    #[error("execution error: {reason}")]
-    Execution { reason: String },
+    #[error("{error}")]
+    Execution {
+        error: Box<deno_core::error::JsError>,
```

Comment:
What is the motivation for doing this? Should we couple the `ddsa` with `deno`? (or rather, should we expose deno-specific stuff in the ddsa layer?